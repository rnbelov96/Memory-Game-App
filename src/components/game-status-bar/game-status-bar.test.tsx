import React from 'react';
import renderer from 'react-test-renderer';
import GameStatusBar from './game-status-bar';

describe('GameStatusBar component', () => {
  test('should render game status bar', () => {
    const tree = renderer
      .create(
        <GameStatusBar onStart={() => {}} moves={3} stars={3} timer={20} />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
