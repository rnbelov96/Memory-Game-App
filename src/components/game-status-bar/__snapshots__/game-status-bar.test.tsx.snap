// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameStatusBar component should render game status bar 1`] = `
<section
  className="sc-bdfBQB kIrMDS"
>
  <div>
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-star fa-w-18 "
      data-icon="star"
      data-prefix="fas"
      focusable="false"
      role="img"
      style={Object {}}
      viewBox="0 0 576 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-star fa-w-18 "
      data-icon="star"
      data-prefix="fas"
      focusable="false"
      role="img"
      style={Object {}}
      viewBox="0 0 576 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-star fa-w-18 "
      data-icon="star"
      data-prefix="fas"
      focusable="false"
      role="img"
      style={Object {}}
      viewBox="0 0 576 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
  </div>
  <p>
    00 : 20
  </p>
  <p>
    3 Moves
  </p>
  <svg
    aria-hidden="true"
    className="svg-inline--fa fa-redo fa-w-16 "
    data-icon="redo"
    data-prefix="fas"
    focusable="false"
    onClick={[Function]}
    role="img"
    style={Object {}}
    viewBox="0 0 512 512"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M500.33 0h-47.41a12 12 0 0 0-12 12.57l4 82.76A247.42 247.42 0 0 0 256 8C119.34 8 7.9 119.53 8 256.19 8.1 393.07 119.1 504 256 504a247.1 247.1 0 0 0 166.18-63.91 12 12 0 0 0 .48-17.43l-34-34a12 12 0 0 0-16.38-.55A176 176 0 1 1 402.1 157.8l-101.53-4.87a12 12 0 0 0-12.57 12v47.41a12 12 0 0 0 12 12h200.33a12 12 0 0 0 12-12V12a12 12 0 0 0-12-12z"
      fill="currentColor"
      style={Object {}}
    />
  </svg>
</section>
`;
